keps/sig-api-machinery/2896-openapi-v3/README.md

```diff
@@ -673,6 +723,8 @@ some monitoring details). For now, we leave it here. -->
 
 ###### How does this feature react if the API server and/or etcd is unavailable?
 
+The feature is part of the API server and will not function if it is unavailable.
```

Comment:
it does not depend on etcd though. So I think we will continue to serve in case etcd is unstable. Whether this has value is another question.